views/components/ScrollContainer.svelte

```diff
@@ -96,10 +124,19 @@
               Image n+1 is below and becomes visible
               => smooth transition n to n+1
       -->
-      {#each imageUrlsReverse as { id, image }}
+      {#each imageUrlsReverse as { id, image }, i}
         {#if image && [index - 1, index, index + 1].includes(id)}
+          <!-- 
+            Fade out animation is skipped if the appearing image has not the same aspect ratio.
+            Instead the appearing image is faded in.
+            Otherwise the fading image changes dimensions because it applies the updated 'imageHeight' of the appearing image during fade out.
+          -->
           <img
             class="q-scroll-graphic-image"
+            class:q-scroll-graphic-image--transition-animation={item.transitionAnimation ===
```

Comment:
This default value of this option is `false`. I would not recommend to fallback to `true` if the value is not set (older scroll-graphics). Did you discuss this with Adina?- Does she think that all existing scroll-graphics should have this option enabled?- If so I would rather recommend to have an option which allows to disable the animation (like in the imageslider tool). If not I would recommend to selectively enable this option for existing scroll-graphics.